avail-cli/src/main/kotlin/com/avail/tools/compiler/Compiler.kt

```diff
@@ -131,12 +132,14 @@ import java.util.concurrent.locks.ReentrantLock
  *
  * @author Todd L Smith &lt;todd@availlang.org&gt;
  * @author Leslie Schultz &lt;leslie@availlang.org&gt;
+ * @author Richard Arriaga &lt;rich@availlang.org&gt;
  */
 object Compiler
```

Comment:
Just tested it and made a small change to get it pointing to repositories. This will be fixed up when I address make the repositories directory home relative per another comment in this review.

Reply:
And to answer your original question, it works.